app/views/content/non-uk-teachers/why-train-to-teach-transcript.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Why train to teach in England?"
+description: |-
+  Full transcript for the 'Why train to teach in England?' video.
+---
+
+
+###  Why train to teach in England? 
+
+**Christobel Gondwe, Maths teacher:**
+If you want a job that you will love and you will feel wanted, and you will be looked after in essence, and you want a job where you can develop and progress, I’d definitely say to consider teaching in the UK. 
+
+### Is it hard to get on to a teacher training course in England? 
+
+**Nellie D’Costa, Trainee science teacher:**
+I assumed that it would be hard because I come from India. But then it was quite straightforward and easy. 
+
+**Giorgio Tatananni, Trainee modern foreign languages teacher:**
+Getting a training place here was so much easier than I thought. 
+
+### How about sorting your visa and documents? 
+
+**Christobel Gondwe, Maths teacher:**
```

Comment:
I think once we've said their job role once, we shouldn't repeat it - so from here down, just the name?

Otherwise it will get really annoying for screen reader users?